versions/3.1.1.md

```diff
@@ -2990,13 +3008,11 @@ MyResponseType:
       monster: https://gigantic-server.com/schemas/Monster/schema.json
 ```
 
-Here the discriminator _value_ of `dog` will map to the schema `#/components/schemas/Dog`, rather than the default (implicit) `#/components/schemas/dog`.  If the discriminator _value_ does not match an implicit or explicit mapping, no schema can be determined and validation SHOULD fail. Mapping keys MUST be string values, but tooling MAY convert response values to strings for comparison.
+Here the discriminator property _value_ of `dog` will map to the schema `#/components/schemas/Dog`, rather than the default (implicit) value of `#/components/schemas/dog`.  If the discriminator property _value_ does not match an implicit or explicit mapping, no schema can be determined and validation SHOULD fail.
```

Comment:
Given that the definition of `propertyName` also uses  "discriminator value", I'd remove the "property" here, same as in 3.0.4:

```suggestion
Here the discriminator _value_ of `dog` will map to the schema `#/components/schemas/Dog`, rather than the default (implicit) value of `#/components/schemas/dog`.  If the discriminator _value_ does not match an implicit or explicit mapping, no schema can be determined and validation SHOULD fail.
```

Reply:
I'd rather fix the definiton of `propertyName` as I found this wording very confusing.  To me, the discriminator value is the value of the `discriminator` field, which is how we'd talk about any other field.

Reply:
Whatever works better for you, as long as it is consistent between 3.0.4 and 3.1.1.

Reply:
@ralfhandl

> as long as it is consistent between 3.0.4 and 3.1.1.

See my previous comment:

> Once this PR is accepted, I will backport jdesrosiers 's changes, as merged here, to 3.0.4 and include a co-author credit (unless you'd rather do it yourself, Jason).

which is why I don't want to do it piecemeal- I'm mentally tracking about 30 issues and being systematic about how I synchronize them across the various releases.  I made a change on 3.0.4, Jason made a change on 3.1.1.  This is me forward-porting my 3.0.4 changes, merging them, and then I'll backport the merge result to ensure Jason's changes make it to 3.0.4.

Reply:
OK I have [switched to "discriminating value"](https://github.com/OAI/OpenAPI-Specification/pull/3846/commits/dd44da96dbca2aa92d4c0efe87448accfd213ed0) across the whole PR, as I agree with @ralfhandl that what i had was confusing, but I also find "discriminator property" to be confusing as it means "value of the `discriminator` property in the OAD" to me, rather than value of the property nmaed by `#/.../discriminator/propertyName`.

@ralfhandl If that doesn't feel right, I would also be open to "discriminating property".  The important part for me is distinguishing between "discriminator" (which to me is the `discriminator` keyword and Discriminator Object) and "discriminating" (which to me implies the instance property and value).